python/lsst/ts/scheduler/driver.py

```diff
@@ -10,6 +13,80 @@
 __all__ = ["Driver", "DriverParameters"]
 
 
+class DriverTarget(Target):
+    def __init__(self, targetid=0, fieldid=0, band_filter="",
+                 ra_rad=0.0, dec_rad=0.0, ang_rad=0.0,
+                 num_exp=0, exp_times=[]):
+        """Initialize the class.
+
+        Parameters
+        ----------
+        targetid : int
+            A unique identifier for the given target.
+        fieldid : int
+            The ID of the associated OpSim field for the target.
+        band_filter : str
+            The single character name of the associated band filter.
+        ra_rad : float
+            The right ascension (radians) of the target.
+        dec_rad : float
+            The declination (radians) of the target.
+        ang_rad : float
+            The sky angle (radians) of the target.
+        num_exp : int
+            The number of requested exposures for the target.
+        exp_times : list[float]
+            The set of exposure times for the target. Needs to length
+            of num_exp.
+        """
+        super().__init__(targetid=targetid, fieldid=fieldid, band_filter=band_filter, ra_rad=ra_rad,
+                         dec_rad=dec_rad, ang_rad=ang_rad, num_exp=num_exp, exp_times=exp_times)
```

Comment:
This `__init__` only calls the parent class `__init__`. If you think that is likely to be so for the long run then I suggest you omit this method method entirely (and refer the user to the parent class in the class doc string for the constructor parameters).